tests/unit/test_aws.py

```diff
@@ -1,2 +1,50 @@
-def test_fake():
-    pass
+import pytest
+from mock import MagicMock
+
+from proxysql_tools.aws import get_network_interface, get_network_interface_state, network_interface_attached, \
+    detach_network_interface
+
+
+def test__get_network_interface(mocker):
+    mock_boto3 = mocker.patch('proxysql_tools.aws.boto3')
+    mock_ec2 = MagicMock()
+    mock_boto3.client.return_value = mock_ec2
+    get_network_interface('0.0.0.0')
+    mock_boto3.client.assert_called_once_with('ec2')
+
+
+def test__get_network_interface_state(mocker):
+    mock_boto3 = mocker.patch('proxysql_tools.aws.boto3')
+    mock_ec2 = MagicMock()
+    mock_boto3.client.return_value = mock_ec2
+    get_network_interface_state('some interface')
+    mock_boto3.client.assert_called_once_with('ec2')
+
+
+def test__network_interface_attached_if_network_state_is_attached(mocker):
```

Comment:
Rename the unit test function to `test__network_interface_state_returns_true_if_attached`